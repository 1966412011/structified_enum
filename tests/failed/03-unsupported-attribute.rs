use structified_enum::structify;

#[structify]
#[other_attr]
enum Foo {}

#[structify]
enum Bar {
    #[other_attr]
    A
}

fn main() {}