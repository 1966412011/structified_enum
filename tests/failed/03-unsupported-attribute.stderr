error: unsupported attribute
 --> tests/failed/03-unsupported-attribute.rs:4:1
  |
4 | #[other_attr]
  | ^

error: unsupported attribute
 --> tests/failed/03-unsupported-attribute.rs:9:5
  |
9 |     #[other_attr]
  |     ^
